plugins/dataTypes/Boolean/schema.json

```diff
@@ -0,0 +1,11 @@
+{
+	"title": “Boolean”,
```

Comment:
You where right. I cannot find out how it happend. Maybe I used an other editor then my default.
Quotes are fixed now.